### PR TITLE
Update Install Astro on AWS using a cross-account role

### PR DESCRIPTION
**Pull request by kwcanuck**
[Google Doc](https://docs.google.com/document/d/1l9rpsH0W_moWWKBRDBPq3LTJ2VVvMwozJpF87kdO-3M/edit#) that prompted the edits in this PR.

### CODE REVIEW

File: astro/cross-account-role-setup.md [hunk 1]
```diff
@@ -210,13 +210,14 @@ To monitor changes to the cross-account role policy and permissions boundaries,
...
+After creating the AWS account, provide Astronomer with the following information:
 
 - Your AWS Account ID.
 - Your preferred Astro cluster name.
 - The AWS region that you want to host your cluster in.
 - Your preferred node instance type.
 - Your preferred maximum node count.
+- An instance type for the Airflow metadata database.
 
 If you do not specify configuration preferences, Astronomer creates a cluster with `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1`. For information on all supported regions, configurations, and defaults, see [AWS cluster configurations](resource-reference-aws.md).
```
Comment (dnarain-astro) [inline astro/cross-account-role-setup.md:222]: Should also specify the default for the metadata database since we are asking for the instance type.

**Review by dnarain-astro — APPROVED** [review]
Looks good, added a comment about documenting the RDS default, but other than that, this all looks good.

**Comment by kwcanuck** [comment]
Clarification request sent to @dnarain-astro.

File: astro/cross-account-role-setup.md (hunk 1 above)
Comment (chomipi88) [reply astro/cross-account-role-setup.md:222]: Agreed ^

Comment (kwcanuck) [reply astro/cross-account-role-setup.md:222]: As discussed I updated the statement to include the default RDS value.

**Pull request merged**